var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -335,6 +340,11 @@ def _get_sys_type(self, spec):
             sys_type = env["SYS_TYPE"]
         return sys_type
 
+    @property
+    def libs(self):
+        shared = True if "+shared" in self.spec else False
+        return find_libraries("liblbann", root=self.prefix, shared=shared, recursive=True)
```

Comment:
FWIW this looks like it matches the default implementation of `libs`, so I'm curious if you were having issues with that. No problem with redefining it here though.

Reply:
@scheibelp Hmm, I am trying to remember why I had to add that.